WordPress/WordPressShareExtension/ShareModularViewController.swift

```diff
@@ -214,17 +214,7 @@ class ShareModularViewController: ShareExtensionAbstractViewController {
 
 extension ShareModularViewController {
     fileprivate func dismiss() {
-        // In regular width size classes (iPad), action extensions are displayed
-        // in a small modal, which looks strange when this VC is dismissed
-        // before the main / presenting controller with its white background.
-        // This workaround simply dismisses the modular VC along with the main extension VC.
-        // See https://github.com/wordpress-mobile/WordPress-iOS/issues/8646 for more info.
-        guard UIDevice.isPad() == false && originatingExtension != .saveToDraft else {
```

Comment:
As far as I can tell, the share UI has changed since this was originally done and this work around is no longer needed.

Reply:
Right, I'm seeing the extension always displayed in a large modal, not a small one.